core/src/main/java/com/graphhopper/routing/util/parsers/BikeCommonAverageSpeedParser.java

```diff
@@ -160,7 +160,8 @@ else if (way.hasTag("bicycle", "yes"))
             speed = PUSHING_SECTION_SPEED; // unknown surface
         } else if (way.hasTag("service")) {
             speed = highwaySpeeds.get("living_street");
-        } else if ("track".equals(highwayValue)) {
+        } else if ("track".equals(highwayValue) ||
+                   "bridleway".equals(highwayValue) ) {
```

Comment:
```suggestion
        } else if ("bridleway".equals(highwayValue) && surfaceSpeed != null ) {
              speed = surfaceSpeed;
        } else if ("track".equals(highwayValue) {
```

Otherwise, we might take into account a track type that was mistakenly assigned to the bridleway.

Reply:
Good catch! Thank you!

Reply:
I was wrong, your suggested changes do not compile. And this test already passes :
```
        way.clearTags();
        way.setTag("highway", "bridleway");
        way.setTag("surface", "gravel");
        way.setTag("bicycle", "designated");
        way.setTag("tracktype", "grade5");
        assertPriorityAndSpeed(PREFER, 12, way);
```
@otbutz What did you mean?

Reply:
Could you test the following?

```java
        way.clearTags();
        way.setTag("highway", "bridleway");
        way.setTag("bicycle", "designated");
        way.setTag("tracktype", "grade5");
        assertPriorityAndSpeed(PREFER, 12, way);
```

Reply:
> Could you test the following?
> 
> ```java
>         way.clearTags();
>         way.setTag("highway", "bridleway");
>         way.setTag("bicycle", "designated");
>         way.setTag("tracktype", "grade5");
>         assertPriorityAndSpeed(PREFER, 12, way);
> ```

This test fails as the speed result is 4km/h, which is the assigned trackTypeSpeed for 'grade5'. This result is fine from my perspective.

Reply:
I was under the impression that this tag shouldn't be paired with a bridleway, so I went the extra mile and fired up Overpass Turbo. As it turns out, tagging the tracktype for a bridleway is legit and actually quite common:

![grafik](https://github.com/user-attachments/assets/6fa7b62c-67d8-4daf-8784-c624c84458ce)